### ISSUE TITLE
macOS Getting Started Issues on M1 Pro

### ISSUE THREAD

**Issue by razerbeans** [elk-audio/elk-docs#2]
Hey there,

tl;dr - Documentation for macOS points to an older version of Sushi (v1.0.0) and should probably be updated (v1.1.0).

I've noticed that I've hit a bit of a snag when following the documentation provided for getting started on macOS. I'm not sure if I've done something wrong at some point, but here's what I am seeing:

The guidance on the following doc https://elk-audio.github.io/elk-docs/html/intro/getting_started_with_development_kit_software.html is to download the latest sushi release, unzip, and run the following command after setting attributes: `./sushi --coreaudio -c config_files/play_brickworks_synth.json`

Two issues I'm having with this command:
1. Coreaudio is not an available flag for sushi and fails
2. `play_brickworks_synth.json` doesn't exist in the config_files as part of the release.

Digging further, it looks like the link on the documentation page directs to sushi release v1.0.0 explicitly, but the most recent available is v1.1.0. After downloading v1.1.0, I can see that the relevant config files are there. So it looks like the link to the release is outdated.

I _am_ getting a failure when attempting to run the supplied command on my own machine, even with the files there, but it looks like library dependencies and is probably a separate issue.

**Comment by nyboer** [comment]
I just came here to document the same thing, so I'll just a voice of support!
